templates/limit/templates.yaml

```diff
@@ -0,0 +1,147 @@
+dataset: limit
+templates:
+  0898caf1-f3e4-493f-a838-84a678176c14: !Template
+    id: 0898caf1-f3e4-493f-a838-84a678176c14
+    jinja: '{{sentence}}
+
+
+      What is the last moving entity mentioned in the sentence if any?
+
+      |||
+
+      {% if (motion_entities  | length) > 0 %}
+
+      {{ (motion_entities | sort(attribute="start_index") | last)["entity"] }}
+
+      {% endif %}'
```

Comment:
add else condition to say "no entities in motion" if none

Reply:
Added. For other prompts with a similar possible answer, I literally told what to output in case there's no entity (i.e "Return no entity if there's none" in the prompt). See `find_entities_question` below.
Do you think it's desirable or not?